puppet/modules/web/templates/deploy-stagingyum.sh.erb

```diff
@@ -9,3 +9,5 @@
 
   # Publish the repo - stderr/out redirect is required to stop the noninteractive shell from hanging
   rsync --recursive --times --verbose --one-file-system --delete-after <%= @home %>/rsync_cache/$PROJECT/$RELEASE <%= @yum_directory %>/$PROJECT/ 2>&1 >/dev/null ;
+  # Cleanup - no need to keep the rpms
+  find <%= @home %>/rsync_cache/$PROJECT/$RELEASE -iname '*.rpm' -delete || true
```

Comment:
this is taken from the Debian cleanup. technically we could also yank the whole folder here with an `rm -rf`, what's you preference?

Reply:
I would have thought we do not delete the cache so that we do not have to re-sync things that have already been synced and saved time and cost.

Reply:
Well, today I had to delete things to be able to upload new things.
Like, there was a full 3.9 that we never will publish again in the cache.

Reply:
Well, today I had to delete things to be able to upload new things.
Like, there was a full 3.9 that we never will publish again in the cache.

Reply:
Should it become part of retiring a release? Make it part of a procedure?

Reply:
It certainly could, even tho I don't like that too much as such things tend to get forgotten.